lib/queries/getChallenge.js

```diff
@@ -1,6 +1,17 @@
 'use strict';
 
-module.exports = function (id) {
-  return 'SELECT id, title, description, date, org_id, creator_id, active '
-    + 'FROM challenges WHERE id = ' + id;
+module.exports = function (challengeId) {
+  return 'SELECT '
+    + 'challenges.id AS chal_id, '
+    + 'challenges.title AS chal_title, '
+    + 'challenges.description AS chal_desc, '
+    + 'tags.name AS tags_name, '
+    + 'org_id, tags_id '
+    + 'FROM '
+    + 'challenges '
+    + 'JOIN tags_challenges ON tags_challenges.challenges_id = challenges.id '
+    + 'JOIN tags ON tags_challenges.tags_id = tags.id '
+    + 'WHERE '
+    + 'tags.active = true AND challenges.active = true '
```

Comment:
```tags.active = true``` Do we need to filter the inactive tags belonging to a challenge here or do we want to return all the tags and display them in a different format (different colour?) on the view?